Gemfile

```diff
@@ -29,7 +28,7 @@ gem 'jbuilder', '~> 2.5'
 # gem 'redis', '~> 3.0'
 # Use ActiveModel has_secure_password
 # gem 'bcrypt', '~> 3.1.7'
-
+gem 'rubocop', require: false
```

Comment:
This gem should go into the development group.

https://github.com/oscarjosecostero/Easyrent/blob/ec2b4aecf5f5d5f82a8f99609b8cef2959422c23/Gemfile#L44